patches/html-to-image+1.6.2.patch

```diff
@@ -0,0 +1,168 @@
+\ No newline at end of file
+diff --git a/node_modules/html-to-image/lib/index.js b/node_modules/html-to-image/lib/index.js
+index b18a4be..3ffa079 100644
+--- a/node_modules/html-to-image/lib/index.js
++++ b/node_modules/html-to-image/lib/index.js
+@@ -1,3 +1,4 @@
++alert('lib');
```

Comment:
This should be removed

Reply:
Nice catch, I thought nobody is looking at those messy patch files 😁